src/prpy/base/mico.py

```diff
@@ -99,4 +109,11 @@ def Servo(self, velocities):
             self.GetRobot().GetController().Reset(0)
             self.servo_simulator.SetVelocity(velocities)
         else:
-            raise NotImplementedError('Servo is not implemented.') 
+            self.SendVelocitiesToMico(velocities)
+            #reset watchdog timer
+            self.servo_watchdog.reset()
+
+
+    def SendVelocitiesToMico(self, velocities):
```

Comment:
This is so the watchdog can call something without going through Servo. If I had it in servo, and the watchdog called Servo, then it would reset itself